linux_os/guide/system/software/integrity/software-integrity/aide/package_aide_installed/rule.yml

```diff
@@ -31,6 +31,7 @@ references:
     iso27001-2013: A.11.2.4,A.12.1.2,A.12.2.1,A.12.4.1,A.12.5.1,A.12.6.2,A.14.1.2,A.14.1.3,A.14.2.2,A.14.2.3,A.14.2.4,A.14.2.7,A.15.2.1,A.8.2.3
     cis-csc: 1,11,12,13,14,15,16,2,3,5,7,8,9
     anssi: NT28(R51)
+    srg: SRG-OS-000363-GPOS-00150
```

Comment:
I disagree.  The requirement is to "notify designated personnel if baseline configurations are changed in an unauthorized manner."  This would cover other unauthorized means beyond someone logging in and changing something in an audited manner.  AIDE has been included for a few RHEL releases and should continue to be included in the OSPP/STIG profile.

Reply:
Audit can only tell that a file was opened with write access. It cannot tell you if the application made any change. Also, OSPP makes it mandatory to ship the logs off of the system, therefore there is no requirement on how much logs are required to be kept on end points. So, in the face of uncertainty of whether you have enough log data to guess if a file changed, it's much easier to snapshot a system with aide and check system integrity at a future time.